File: Frontend/voitheia-App/src/app/common/models/requestDTO.ts

export class RequestDTO {
    public type: string;
    public lastName: string;
    public firstName: string;
    public email: string;
    public phoneNumber: string;
    public street: string;
    public zip: string;
    public city: string;
    public description: string;

    constructor() {
        this.type = "";
        this.lastName = "";
        this.firstName = "";
        this.email = "";
        this.phoneNumber = "";
        this.street = "";
        this.zip = "";
        this.city = "";
        this.description = "";
    }
}

File: Backend/ActiveCruzer/Models/DTO/Request/RequestDto.cs

namespace ActiveCruzer.Models.DTO.Request
{
    public class RequestDto : MinimalRequestDto
    {
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string Street { get; set; }
        public string Zip { get; set; }
        public string City { get; set; }

    }
}
